sunpy/__init__.py

```diff
@@ -54,17 +49,10 @@ def _get_bibtex():
 
 
 __citation__ = __bibtex__ = _get_bibtex()
-
```

Comment:
I feel like removing *all* this whitespace makes it less readable, single space between different bits would be clearer to me, but whatever.

Reply:
Wrong.

Reply:
Fixed.